app/components/candidate_interface/new_references_needed_component.html.erb

```diff
@@ -1,7 +1,13 @@
 <div class="app-banner app-banner--info" aria-labelledby="info-message">
   <div class="app-banner__message">
     <h2 class="govuk-heading-m" id="info-message">
-      <% if reference_status.number_of_references_that_currently_need_replacing == 1 %>
+      <% if reference_status.not_requested_yet?.count == 1 %>
+        <p>You have not confirmed your referee yet.</p>
+        <p><%= govuk_link_to 'Confirm your referee so that they can be contacted', candidate_interface_confirm_additional_referees_path %></p>
+      <% elsif reference_status.not_requested_yet?.count > 1 %>
+        <p>You have not confirmed your referees yet.</p>
+        <p><%= govuk_link_to 'Confirm your referees so that they can be contacted', candidate_interface_confirm_additional_referees_path %></p>
```

Comment:
I think this is fine. Another option would be to do this in the component:

`def pluralize_referee
    'referee'.pluralize(reference_status.not_requested_yet?.count)
  end
`

Then you can just interpolate like this.

`if reference_status.not_requested_yet?.present?
    You have not confirmed your <%=pluralize_referee %> yet.
    <%= govuk_link_to 'Confirm your <%= pluralize_refereee %> so that they can be contacted', 
     candidate_interface_confirm_additional_referees_path %>
 end 
`